### ISSUE TITLE
Comprei o serviço 2 vezes 

### ISSUE THREAD

**Issue by Emmanuel20087** [idolize/xcloud-keyboard-mouse#114]
Eu comprei porque eu gosto de jogar no teclado porém no meu cartão consta que eu paguei 2 vezes ou seja 44 reais oque da 8 dólares ao invés de 4 dólares eu queria que pudesse devolver ou 4 dólares que por mais que não pareça é muito no Brasil que é onde eu moro mas eu queria continuar com o serviço então seria 4 dólares pra continuar o serviço e devolver meus outros 4 dólares se não puder me avise se puder me avise estarei esperando.

![16753715059755556544074755334839](https://user-images.githubusercontent.com/124309282/216448147-389f34f1-82c2-43e7-80b9-0c653078c05c.jpg)

![16753714830481917852328935798309](https://user-images.githubusercontent.com/124309282/216448028-26c8bbdf-f14b-45c3-92b7-dc123cf3e768.jpg)

**Comment by idolize** [comment]
Hello! If you can tell me the email address(es) you used to pay I can double check - but it should only be possible to pay once for a given email address.

**Comment by Emmanuel20087** [comment]
My gmail is Emmanuelgames2008@gmail.com

**Comment by idolize** [comment]
I only see one charge for that email - are you certain it was charged twice if you check your statement again?

**Comment by Emmanuel20087** [comment]
yes there really are 2 charges but for some reason it was done before I actually bought it because I received the money precisely to buy the extension but I'll see what I can do here since the value has decreased for some reason.
![Screenshot_20230203-161426](https://user-images.githubusercontent.com/124309282/216688337-44d7d769-877f-4020-86fd-8814d998f602.jpg)


I paid before I received the money I was going to use to pay for some reason

**Comment by idolize** [comment]
Okay - I'm sorry about that! Not sure what the issue is. If you can't figure it out let me know and I can issue a refund.

**Comment by Emmanuel20087** [comment]
not at all I don't regret anything even if I spent more for lifetime access it's even cheap 8 dollars thank you very much for this xcloud project but I don't need a refund.
